app/views/announcements/index.html.erb

```diff
@@ -1,30 +1,26 @@
 <%= render "layouts/view_header", resource: @announcements.first %>
 
-<div class="section-detail">
-  <table class="table table-hover table-curved table-condensed">
-    <thead>
-      <tr>
-        <th>Name</th>
-        <th>Description</th>
-        <th>Published?</th>
-        <th>Edit</th>
-      </tr>
-    </thead>
-    <tbody>
-      <% @announcements.each do |announcement| %>
-        <tr>
-          <td>
-            <strong><%= announcement.name %></strong>
-          </td>
-          <td>
-            <strong><%= announcement.description&.truncate(20) %></strong>
-          </td>
-          <td class="has-text-centered"><%= yes_no(announcement.published?) %></td>
-          <td><%= edit_button(announcement) %></td>
-        </tr>
+<div class="row">
+  <% @announcements.each do |announcement| %>
```

Comment:
If this is not really a `<table>`, is it actually an `<ul>` instead?

Reply:
Love that idea!